src/PaymentStates.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Webhook Module for PHP
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Webhook;
+
+final class PaymentStates
+{
+    public const STATE_OPEN = 'open';
```

Comment:
Could you provide some explanation to the states that which means what? (I know some of them are pretty much self explanatory but for example what is the difference between `open` and `in_progress`?

Reply:
I think `open` is indeed out of place, it is not used and it's irrelevant for the purposes of procesing notifications, I was just copying from shopware's order transaction states, I will just remove it and then the rest are self explanatory.